controllers/leads.js

```diff
@@ -23,8 +22,10 @@ exports.create = function(request, response) {
       state: request.body.FromState,
       callerName: request.body.CallerName
     });
-
-    return newLead.save();
+    return newLead.save()
+    .then(function() {
+      response.send(twiml.toString());
+    })
```

Comment:
I don't want to be picky, but looks like you're missing a semicolon here. :smirk: